examples/web_apps/webxr_hand_demo/js/physics_worker_setup.js

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+/* global FS, importScripts, PhysicsWorker */
+
+function preloadFunc(url) {
```

Comment:
As far as I know, there is no easy way to get this function other than copy paste. This is located in utils.js but as a module export, but since physics_worker_setup.js cannot be a module file, it cannot use that function. So I just copy pasted it here instead.

Reply:
You can use Webpack (which we already use for bundling the JS) to copy and paste it the worker as an additional target. That would be the best way of doing it @ldcWV . I think you can also use requires / imports across webworkers.

Reply:
You can also just eval / import other JS scripts to prevent duplicating the function.

Reply:
>You can use Webpack
I'd prefer to avoid Webpack for this example app. It adds complexity and makes iteration slower (due to build time). This is just an example app so optimizing browser load-time isn't a priority.

Reply:
> You can also just eval / import other JS scripts to prevent duplicating the function.

Hey, I'm not sure what you mean by this. Do you mind elaborating?

As far as I know, `eval` is a function that takes in JS code as a string and executes it. Are you suggesting there is some way to get this code as a string without copy paste?

Reply:
Yeah, the code can be hosted remotely too.

Reply:
Wouldn't that require duplication anyways? The issue is that this `preload` function is already in `utils.js` but I cannot use it because it has an `export`. I don't see how making another JS file would resolve this.